cp23/hpi-fun2/motivation/notes.md

```diff
@@ -0,0 +1,46 @@
+# Motivation
+
+The problem is about finding the path coverage of the tree. The problem should find the number of nodes, whos path lengths exceed the given threshold (including all its decendants). Also, it requires to find the number of paths to leafs.
```

Comment:
To be honest, I am not sure how to formalize that, since that is only an additional condition, that tunes the final solution. We could still reduce the problem to path coverage. What would be a proper formalization here?